src/test/java/com/code_intelligence/jazzer/junit/CorpusDirectoryTest.java

```diff
@@ -0,0 +1,183 @@
+            // Add corpus directory as initial libFuzzer parameter.
+            .configurationParameter("jazzer.internal.arg.0", "fake_test_argv0")
+            .configurationParameter(
+                "jazzer.internal.arg.1", corpusDirectory.toAbsolutePath().toString())
+            .execute();
+
+    results.containerEvents().assertEventsMatchExactly(event(type(STARTED), container(ENGINE)),
+        event(type(STARTED), container(uniqueIdSubstrings(ENGINE, CLAZZ))),
+        event(type(STARTED), container(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ))),
+        event(type(FINISHED), container(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ)),
+            finishedSuccessfully()),
+        event(type(FINISHED), container(uniqueIdSubstrings(ENGINE, CLAZZ)), finishedSuccessfully()),
+        event(type(FINISHED), container(ENGINE), finishedSuccessfully()));
+
+    results.testEvents().assertEventsMatchExactly(
+        event(type(DYNAMIC_TEST_REGISTERED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ))),
+        event(type(STARTED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ, INVOCATION + 1))),
+        event(type(FINISHED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ, INVOCATION + 1)),
+            displayName("<empty input>"), finishedSuccessfully()),
+        event(type(DYNAMIC_TEST_REGISTERED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ))),
+        event(type(STARTED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ, INVOCATION + 2))),
+        event(type(FINISHED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ, INVOCATION + 2)),
+            displayName("seed"), finishedSuccessfully()),
+        event(type(DYNAMIC_TEST_REGISTERED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ))),
+        event(type(STARTED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ, INVOCATION + 3)),
+            displayName("Fuzzing...")),
+        event(type(FINISHED), test(uniqueIdSubstrings(ENGINE, CLAZZ, INPUTS_FUZZ, INVOCATION + 3)),
+            displayName("Fuzzing..."),
+            finishedWithFailure(instanceOf(FuzzerSecurityIssueMedium.class))));
+
+    // Crash file should be emitted into the artifacts directory and not into corpus directory.
+    assertCrashFileExistsIn(artifactsDirectory);
+    assertNoCrashFileExistsIn(baseDir);
+    assertNoCrashFileExistsIn(corpusDirectory);
+    assertNoCrashFileExistsIn(generatedCorpus);
+
+    // Verify that corpus files are written to given corpus directory and not generated one.
+    assertThat(Files.list(corpusDirectory)).isNotEmpty();
+    assertThat(Files.list(generatedCorpus)).isEmpty();
+  }
+
+  @Test
+  public void fuzzingDisabled() throws IOException {
+    assumeTrue(System.getenv("JAZZER_FUZZ").isEmpty());
+
+    Path corpusDirectory = baseDir.resolve(Paths.get("corpus"));
+    Files.createDirectories(corpusDirectory);
+    // noinspection ResultOfMethodCallIgnored
+    corpusDirectory.resolve("corpus_entry").toFile().createNewFile();
```

Comment:
Would `Files.createFile()` work here?  We could stay in `Path` land that way.

Reply:
Yes! Changed.